MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -783,6 +780,33 @@ bool ancestorInList(GameObject go)
 			}
 		}
 
+		private Guid GenerateObfuscatedUserId(IHostAppUser hostAppUser)
+		{
+			if (GrantedPermissions.HasFlag(Permissions.UserTracking) && GlobalAppId != string.Empty)
```

Comment:
Would be good to include a null test here too.

Reply:
We do guarantee that is not null at the time that the constructor is called, just so that we don't have to null check this value everywhere it is read from.

---
In reply to: [453088588](https://github.com/microsoft/mixed-reality-extension-unity/pull/216#discussion_r453088588) [](ancestors = 453088588)